libs/platform/src/lib/components/dynamic-page/dynamic-page.component.html

```diff
@@ -1,5 +1,5 @@
 <section class="fd-dynamic-page" [attr.aria-label]="ariaLabel">
-    <header class="fd-dynamic-page__header-fixed" #header>
+    <header class="fd-dynamic-page__header-fixed" [class]="{'fd-dynamic-page__header--not-collapsible': !headerCollapsible}" #header>
```

Comment:
I don't see this class's styles getting applied in the example: 
![image](https://user-images.githubusercontent.com/53509521/98761517-9811d100-23fb-11eb-82f0-0abe8b951ea6.png)

Reply:
This is because the style works on the title area not on the page header element itself:

<img width="1680" alt="Screen Shot 2020-11-13 at 7 50 21 AM" src="https://user-images.githubusercontent.com/48103491/99091695-1352be80-2585-11eb-941a-4cf18b92cfbf.png">

And it is only active on 'hover' or 'focus' of the title area.